### PR TITLE
Fix Multiple instances of Object Bucket, Object Bucket Claims under storage section

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2087078
Signed-off-by: Gowtham Shanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Review by bipuladh — APPROVED** [review]
/retest

**Comment by bipuladh** [comment]
Please adjust the position of data foundation as well

File: frontend/packages/console-app/console-extensions.json [hunk 1]
```diff
@@ -63,10 +63,10 @@
   {
     "type": "console.navigation/resource-cluster",
     "properties": {
-      "id": "volumesnapshots",
+      "id": "volumesnapshotcontents",
       "section": "storage",
       "name": "%console-app~VolumeSnapshotContents%",
-      "insertBefore": "deployments",
+      "insertAfter": "volumesnapshotclasses",
```
Comment (bipuladh) [inline frontend/packages/console-app/console-extensions.json:69]: @spadgett needs approval

**Review by bipuladh — APPROVED** [review]

**Review by spadgett — APPROVED** [review]
/approve

**Pull request merged**